src/Microsoft.AspNetCore.Sockets.Client.Http/Utils.cs

```diff
@@ -42,4 +43,10 @@ internal static Uri AppendQueryString(Uri url, string qs)
             return builder.Uri;
         }
     }
+
+    public class Constants
+    {
+        private static readonly string DefaultUserAgent = "Microsoft.AspNetCore.Http.Client/1.0.0-alpha";
```

Comment:
Do we want to make it the same as package name (i.e. `Microsoft.AspNetCore.Sockets.Client.Http`)?